src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -822,21 +822,37 @@ def getStatsFromMongoDB(self, detail=False, **kwargs):
         data = {}
         if kwargs.get('rse'):
             data["query"] = 'rse=%s&detail=%s' % (kwargs['rse'], detail)
-            mongoProjection = {"_id": False,
-                               "name": True,
-                               "pfnPrefix": True,
-                               "rucioConMonStatus": True,
-                               "isClean": True,
-                               "timestamps": True,
-                               "counters": True}
-            if detail:
-                mongoProjection["dirs"]  = True
-
-            rseList = kwargs['rse'] if isinstance(kwargs['rse'], list) else [kwargs['rse']]
-            data["rseData"] = []
-            for rseName in rseList:
-                mongoFilter = {'name': rseName}
-                data["rseData"].append(self.msUnmergedColl.find_one(mongoFilter, projection=mongoProjection))
+            allDocs = (kwargs['rse'].lower() == "all_docs") if isinstance(kwargs['rse'], str) else False
+
+            if allDocs:
+                mongoProjection = {
+                    "_id": False,
+                    "name": True,
+                    "isClean": True,
```

Comment:
Don't you want to have `rucioConMonStatus` attribute on this query as well?

Reply:
That's  a good idea  actually. Let me add it into the projection before we merge. And I will also update the commit message.

Reply:
Done